kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/airline/model/Flight.java

```diff
@@ -0,0 +1,62 @@
+package com.kodilla.good.patterns.challenges.airline.model;
+
+
+import java.util.Objects;
+
+public class Flight {
+
+    private String cityStartFlight;
+    private String cityStopFlight;
```

Comment:
lot powinien zawierac startFlight i endFlight. Jesli miasta sa 3 (start, stop, end), to tak na prawde sa to 2 loty